crates/factor-key-value/src/host.rs

```diff
@@ -79,6 +80,7 @@ impl key_value::Host for KeyValueDispatch {}
 
 #[async_trait]
 impl key_value::HostStore for KeyValueDispatch {
+    #[instrument(name = "spin_key_value.open", skip(self), err(level = Level::INFO), fields(otel.kind = "client", kv.backend=self.manager.summary(&name).unwrap_or_default()))]
```

Comment:
If the summary returns `None` do we really `kv.backend=""` or should it be something like `kv.backend="unknown"`?

Reply:
Yep, I like that better.